examples/production/values-export-ingress-nginx.yaml

```diff
@@ -0,0 +1,27 @@
+spiffe-oidc-discovery-provider:
+  tests:
+    tls:
+      enabled: true
```

Comment:
Don't like to expose the testing logic on public API.

Is there a way around this not to do this?

Reply:
The user may have setup things such that tls testing will work (proper certificate (acme) or custom ca loaded in properly). Or a lot of people won't bother and the test will fail without proper config. Even some of our tests don't test it fully. This is the first test that actually does.

Reply:
Should we bother testing the working of TLS? can't we just ignore TLS verification. In other words how much outside of our own scope are we going to test.

Reply:
How do we know our recommendations actually work and we dont break them with PR's? Testing is critical to support IMO. We're going to be recommending some things in the spire-server ingress that will need proper testing too.

I'm good skipping tests of spire itself, as the spire team should be testing those. But things added specifically by the helm chart, that isn't part of spire should probably be tested to ensure it all works together.

Also, the end user may want to test in their QA environment that the particular combination of settings / their cluster all works together to successfully produce a secure endpoint.

Reply:
k8s has a feature in development for that, https://github.com/kubernetes/enhancements/issues/3257 but is not released yet so we can't rely on it. No idea how to solve it without that, other then what I'm doing here. Suggestions welcome.